jenkins/databricks/run-tests.py

```diff
@@ -19,84 +19,25 @@
 import os
 import subprocess
 from clusterutils import ClusterUtils
+import params
 
 
 def main():
-  workspace = 'https://dbc-9ff9942e-a9c4.cloud.databricks.com'
```

Comment:
Make the parameter parsing scripts common for both 'run-build.py' and 'run-test.py'